site/packages/aa-signers/particle/introduction.md

```diff
@@ -0,0 +1,76 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: ParticleSigner
+  - - meta
+    - name: description
+      content: Overview of the ParticleSigner class in aa-signers
+  - - meta
+    - property: og:description
+      content: Overview of the ParticleSigner class in aa-signers
+---
+
+# Particle Signer
+
+`ParticleSigner` is a signer implementation which extends `SmartAccountAuthenticator` to leverage the [Particle SDK](https://github.com/particle/particle-web3-provider). It supports features such as authentication, message and typed data signing, and authentication details retrieval.
```

Comment:
Our web SDK is closed source, I don't believe that's a valid GitHub link; can link to https://www.npmjs.com/package/@particle-network/auth if you'd like